core/storage/exploration/gae_models.py

```diff
@@ -230,8 +230,8 @@ class ExplorationModel(base_models.VersionedModel):
     correctness_feedback_enabled = datastore_services.BooleanProperty(
         default=False, indexed=True)
     # Size in bytes of Android Proto representation.
-    proto_size_in_bytes = datastore_services.IntegerProperty(
-        default=1, indexed=True)
+    _cached_android_proto_size_in_bytes = datastore_services.IntegerProperty(
+        default=None, indexed=True)
```

Comment:
I don't think there should be a default value here? The domain layer should always specify it explicitly.

Reply:
Fixed